pkgs/by-name/st/startup-disk/package.nix

```
@@ -0,0 +1,65 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitLab,
+  nix-update-script,
+  pkg-config,
+  libadwaita,
+  gtk4,
+  glib,
+}:
+rustPlatform.buildRustPackage rec {
```

```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
```

... and use `finalAttrs.version` below accordingly

See https://github.com/NixOS/nixpkgs/issues/293452